manifests/updates/devtron-ingress-override.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace:  devtroncd
 data:
   override: |
-    apiVersion: extensions/v1beta1
+    apiVersion: networking.k8s.io/v1
```

Comment:
keep both with one commented out and a comment to explain which one to use

Reply:
Done